### PR TITLE
TOMEE-3846 Add Comparison page for TomEE v8

### PR DESCRIPTION
**Pull request by sultan**
this is the "per-version" comparison page for v8

### CODE REVIEW

**Comment by sultan** [comment]
ready for review and open to feedbacks

**Comment by sultan** [comment]
commits squashed

**Review by dblevins — CHANGES_REQUESTED** [review]

**Comment by dblevins** [comment]
Sorry, I forget to click "submit" on my PR comments yesterday.

File: docs/comparison.adoc [hunk 1]
```diff
@@ -1 +1,125 @@
...
+== [[specifications]] Detailed list of Jakarta EE 8.0 and MicroProfile 2.0 specifications
+
+Tomee 8.x specific requirement(s) :
+
+* Runs on Java SE 8
+* Specifications below :
+
+[options="header",cols="1,6*^0"]
+|===
+|Specifications|Spec. Version|Tomcat|TomEE WebProfile|TomEE MicroProfile|TomEE Plus|TomEE Plume
```
[the remark this answers is not in the document]
Comment (sultan) [reply docs/comparison.adoc]: two columns merged

File: docs/comparison.adoc [hunk 2]
```diff
@@ -1 +1,125 @@
...
+
+[options="header",cols="1,1"]
+|===
+|Specifications|Implementations included by TomEE
+|Jakarta Annotations, Servlet, Server Pages (JSP), +
+Jakarta Expression Language (EL), WebSocket, +
+Jakarta Authentication (JASPIC), Security, ...|https://tomcat.apache.org/[Apache Tomcat^]
+|Jakarta{nbsp}Standard{nbsp}Tag{nbsp}Library{nbsp}(JSTL)|https://tomcat.apache.org/taglibs.html[Apache Standard Taglib Implementation^]
+|Jakarta Faces (JSF)|https://myfaces.apache.org/[Apache MyFaces^] *(in all TomEE flavors except Plume)* +
+https://projects.eclipse.org/projects/ee4j.mojarra[Eclipse GlassFish Mojarra^] *(in TomEE Plume only)*
```
[the remark this answers is not in the document]
Comment (sultan) [reply docs/comparison.adoc]: replaced

File: docs/comparison.adoc [hunk 3]
```diff
@@ -1 +1,125 @@
...
+|Jakarta Contexts and Dependency Injection (CDI)|https://openwebbeans.apache.org/[Apache OpenWebBeans^]
+|Jakarta Enterprise Beans (EJB)|https://openejb.apache.org/[Apache OpenEJB^]
+|Jakarta Persistence (JPA)|https://openjpa.apache.org/[Apache OpenJPA^] (in all TomEE flavors) +
+https://www.eclipse.org/eclipselink/[EclipseLink^] *(in TomEE Plume only)*
+|Jakarta Transactions (JTA)|Apache{nbsp}Geronimo{nbsp}Transaction{nbsp}Manager
+|Jakarta Mail (JavaMail)|Apache Geronimo JavaMail
+|MicroProfile|Apache Geronimo MicroProfile *(in TomEE 8.x and earlier)* +
+https://smallrye.io/[SmallRye MicroProfile^] *(in TomEE 9.x and later)*
+|Jakarta JSON Binding (JSON-B), +
+Jakarta JSON Processing (JSON-P)|https://github.com/FasterXML/jackson[Jackson^] +
```
[the remark this answers is not in the document]
Comment (sultan) [reply docs/comparison.adoc]: removed

File: docs/comparison.adoc [hunk 4]
```diff
@@ -1 +1,125 @@
...
+https://johnzon.apache.org/[Apache Johnzon^]
+|Jakarta XML Binding (JAXB)|https://projects.eclipse.org/projects/ee4j.jaxb-impl[Eclipse GlassFish JAXB Runtime^]
```
[the remark this answers is not in the document]
Comment (sultan) [reply docs/comparison.adoc]: changed

File: docs/comparison.adoc [hunk 5]
```diff
@@ -1 +1,125 @@
...
+|Web Services|https://cxf.apache.org/[Apache CXF Runtime^]
```
[the remark this answers is not in the document]
Comment (sultan) [reply docs/comparison.adoc]: changed

File: docs/comparison.adoc [hunk 6]
```diff
@@ -1 +1,125 @@
...
+MicroProfile Type-safe Rest Client|||{y}|{y}|{y}
+|Jakarta Authorization, Batch, Connectors, Messaging (JMS), ... +
+Jakarta SOAP Web Services (JAX-WS)||||{y}|{y}
+|Jakarta Faces (JSF) implementation||MyFaces|MyFaces|MyFaces|*Mojarra*
+|Jakarta Persistence (JPA) implementation(s)||OpenJPA|OpenJPA|OpenJPA|OpenJPA, *EclipseLink*
+|===
+
+== [[specifications]] Detailed list of Jakarta EE 8.0 and MicroProfile 2.0 specifications
+
+Tomee 8.x specific requirement(s) :
```
[the remark this answers is not in the document]
Comment (sultan) [reply docs/comparison.adoc]: should be ok now

**Comment by sultan** [comment]
i was able to take into account all these feedbacks, thank you again

**Pull request merged**